aioseop_class.php

```diff
@@ -3335,6 +3347,34 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 		add_filter( "{$this->prefix}display_settings", array( $this, 'filter_settings' ), 10, 3 );
 		add_filter( "{$this->prefix}display_options", array( $this, 'filter_options' ), 10, 2 );
 
+		// This ensures different JS files are enqueued only at the intended screens. Preventing unnecessary processes.
+		$extra_title_len = 0;
+		switch ( $hook_suffix ) {
+			// Screens `post.php` & `./aioseop_class.php` share the same `count-char.js`.
+			case 'post.php' :
+				$info         = $this->get_page_snippet_info();
+				$title        = $info['title'];
+				$title_format = $this->get_title_format( array( 'name' => 'aiosp_snippet' ) );
+
+				if ( ! empty( $title_format ) ) {
+					$replace_title   = '<span id="aiosp_snippet_title">' . esc_attr( wp_strip_all_tags( html_entity_decode( $title ) ) ) . '</span>';
+					$extra_title_len = strlen( $this->html_entity_decode( str_replace( $replace_title, '', $title_format ) ) );
+				}
+			case 'toplevel_page_all-in-one-seo-pack/aioseop_class' :
```

Comment:
when we use grunt in the future, it will complain as there is no comment indicating a fall through from one case to another so add a comment `// fall through` so that it is also obvious to future reviewrs

Reply:
Ok cool. I wasn't sure how to comment on it other than above the 2 cases.